### DISCUSSION TITLE
How do I clean up to save on disk usage?

### DISCUSSION THREAD

**Discussion by iandees** [lima-vm/lima#2552] [Q&A]
Hello! I am on a laptop with relatively low storage. I noticed that `~/.colima/_lima/colima/diffdisk` is 77GB, which is a fairly large percentage of the overall disk usage on my system.

What's the best way to reduce the size of this file?

**Comment by why168** [comment]
https://github.com/lima-vm/lima/discussions/1446

**Reply by iandees** [reply]
I saw that, but it seems to be talking about growing the size of the disk, not shrinking it (and dealing with what happens when you shrink an image and potentially truncate the image).

**Comment by nirs** [comment]
@iandees what is the format of the disk?

You can check using:

```
qemu-img info --force-share ~/.lima/{instance-name}/diffdisk
```

**Reply by iandees** [reply]
I ended up deleting the image to recover storage so I don't have it to check. I'll update if I run into the problem again.

**Reply by nirs** [reply]
If vm provisioning is fully automated, recreating is the simplest way.

**Reply by iandees** [reply]
```
$ qemu-img info --force-share ~/.colima/_lima/colima/diffdisk
image: /Users/iandees/.colima/_lima/colima/diffdisk
file format: qcow2
virtual size: 250 GiB (268435456000 bytes)
disk size: 6.09 GiB
cluster_size: 65536
backing file: /Users/iandees/.colima/_lima/colima/basedisk
backing file format: qcow2
Format specific information:
    compat: 1.1
    compression type: zlib
    lazy refcounts: false
    refcount bits: 16
    corrupt: false
    extended l2: false
Child node '/file':
    filename: /Users/iandees/.colima/_lima/colima/diffdisk
    protocol type: file
    file length: 76.7 GiB (82376196096 bytes)
    disk size: 6.09 GiB
```

This was created using `colima start --cpu 8 --memory 6 --disk 15 --arch aarch64 --vm-type=vz --vz-rosetta`.
